wkcuber/cubing.py

```diff
@@ -195,8 +197,17 @@ def cubing_job(args):
 
 def cubing(source_path, target_path, layer_name, dtype, batch_size, args=None) -> dict:
```

Comment:
The `dtype` attribute is the argument that the user can pass per command line. This is for example `"uint8"`. How to convert this string into an `VALID_VOXEL_TYPE`?

Reply:
There is a `convert_element_class_to_dtype` function in `metadata.py` which should be what you are looking for. Do you know why this wasn't a problem before this PR?